modules/command/src/main/java/org/apache/fluo/command/FluoList.java

```diff
@@ -54,17 +54,37 @@ public static void main(String[] args) throws Exception {
       System.out.println("-----------     ------     ---------");
 
       for (String path : children) {
-        FluoConfiguration appConfig = new FluoConfiguration(config);
-        appConfig.setApplicationName(path);
-        try (FluoAdminImpl admin = new FluoAdminImpl(appConfig)) {
-          String state = "STOPPED";
-          if (admin.applicationRunning()) {
-            state = "RUNNING";
-          }
-          int numWorkers = admin.numWorkers();
-          System.out.format("%-15s %-11s %4d\n", path, state, numWorkers);
-        }
+        listApp(config, path);
+      }
+    }
+  }
+
+  private boolean checkCuratorExists(CuratorFramework curator) {
+    try {
+      return curator.checkExists().forPath("/") != null;
+    } catch (Exception e) {
+      throw new RuntimeException("Error checking if curator exists", e);
```

Comment:
`.forPath` throws Exception and I thought that it was a case were we'd want to show the whole stack trace. I don't think this isn't the cleanest way, so I'm open to suggestions.

Reply:
this seems ok to me.  Could add a comment saying why this was done.   Also its possible that Exception could be a RuntimeException.  Could do.

```java
try {
￼      return curator.checkExists().forPath("/") != null;
   } catch (RuntimeException e) {
      throw e; 
￼    } catch (Exception e) {
      // want stack trace for this exception to show up on command line
￼      throw new RuntimeException("Error checking if curator exists", e);
   }
```